src/components/StakingPage/StakePositionTable/TableComponent/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui';
+import { ITableData } from '../types';
+import styles from '../styles';
+import images from '../../../../images';
+
+const vaultImage = [
+  images.iconVaultArchive,
+  images.iconStaking,
+  images.iconRedeem,
+];
+
+export function vaultIconsRenderer(_: unknown, record: ITableData) {

```

Comment:
I prefer start with render like renderVaultIcons

Reply:
And by the document, I think we should use the first value on easy case, please discuss with me